WordPress/Classes/Services/LoginFacade.m

```diff
@@ -68,17 +71,28 @@ - (void)signInToSelfHosted:(LoginFields *)loginFields
             if ([options objectForKey:@"wordpress.com"] != nil) {
                 [self signInToWordpressDotCom:loginFields];
             } else {
+                NSString *versionString = options[@"software_version"][@"value"];
```

Comment:
Move ahead with this PR but could you get a discussion going internally about this on our preferred P2? We should be consistent across Android and iOS and I'd want us to all chime in. :smile: